packages/webapp/src/components/Form/Unit/index.jsx

```diff
@@ -120,13 +121,20 @@ const Unit = ({
           )}
         </div>
       )}
-
+      {currency && (
+        <div className={styles.currency} data-testid={`${testId}-currency`}>
+          {currency}
+        </div>
+      )}
```

Comment:
Is there a reason you chose to add the currency here? I would add it to line 131, so I'm curious.

Reply:
Yeah, I agree line 131 is better. But essentially I copied `/Form/Input` and on that component all the elements are flat with not as much nesting. The css seems to be based upon the entire element including label.  

So that  allowed me to simply copy and paste, but I can adjust the css so that it works in that position if you would prefer? ...  Using `UpdateEstimatedCropRevenue` as a starting point it looks like ..~1/3 of a pixel out of alignment either way haha (As it is currency symbol looks slightly higher than Input, if I change the css translation from (8,34) to (8,13) it looks slightly lower than Input).

I am happy to change if you prefer.

Reply:
Since you agree, it would be great if you could change.
Thank you Duncan!

Reply:
Changed